cvat-ui/src/utils/copy-to-clipboard.ts

```diff
@@ -0,0 +1,8 @@
+export default function copyToClipboard(str: string): void {
```

Comment:
Let's use the library instead https://github.com/sudodoki/copy-to-clipboard/

Reply:
Done